frontend/lib/src/components/widgets/DataFrame/columns/LinkColumn.ts

```diff
@@ -104,35 +103,45 @@ function LinkColumn(props: BaseColumnProps): BaseColumn {
     sortMode: "default",
     validateInput,
     getCell(data?: any, validate?: boolean): GridCell {
+      if (!data) {
+        return {
+          ...cellTemplate,
+          isMissingValue: true,
+        } as LinkCell
+      }
```

Comment:
`undefined` is currently treated as `null` within all our column implementations since we don't really have any way or reason to represent the difference between `null` vs `undefined` within the underlying dataframe data (there isn't anything comparable within Python/Pandas/Arrow).

Reply:
I also think there isn't really any case currently that an `undefined` is actually passed in, so in theory we could even fully leave out `undefined` ... but I'm not 100% sure on that so handling it this way might be the safer option.